### PR TITLE
Update collection-index.yml

### PR DESCRIPTION
**Pull request by CASL0**
add casl0 devcontainer-features

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -643,4 +643,8 @@
   contact: https://github.com/hsm207/devcontainer-templates/issues
   repository: https://github.com/hsm207/devcontainer-templates
   ociReference: ghcr.io/hsm207/devcontainer-templates
-
+- name: Dev Container Features by CASL0
+  maintainer: CASL0
+  contact: https://github.com/CASL0/devcontainer-features/issues
+  repository: https://github.com/CASL0/devcontainer-features
```
Comment (eljog) [inline _data/collection-index.yml:649]: Thank you for the PR
Looking at your code, where is `apt_get_update_defined here =>
https://github.com/CASL0/devcontainer-features/blob/e26c77df9861a4d47abb28df2415aaf79f836552/src/android-sdk/install.sh#L17C5-L17C19

Comment (CASL0) [reply _data/collection-index.yml:649]: @eljog 

Thank you for your review comment!

It was undefined, so i fix it.

Comment (CASL0) [reply _data/collection-index.yml:649]: https://github.com/CASL0/devcontainer-features/commit/852b7af7476ae652bd3926bca506533ed6084600

**Review by eljog — APPROVED** [review]
Thank you for the contribution to the community

**Pull request merged**